client/src/pages/ScanStore.tsx

```diff
@@ -47,39 +91,259 @@ function ScanStore() {
-              <th>Price</th>
-              <th>Quantity</th>
-            </tr>
-          </thead>
-          <tbody>
-            {shoppingList.map((curItem: CartItem) => (
-              <tr key={curItem.item.barcode}>
-                <td>{curItem.item.name}</td>
-                <td>{curItem.item.price}</td>
-                <td>{curItem.quantity}</td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
+    <Container disableGutters={true} className="ScanStore">
+      <Grid container spacing={1} direction="column" alignItems="stretch">
+        <Grid item xs={12}>
+          <Paper elevation={1}>
+            <StoreHeader storeId={storeID} />
+          </Paper>
+        </Grid>
+        {!showCart && (
+          <Grid item xs={12}>
+            <Paper elevation={1}>
+              <Box p={1}>
+                <Grid item container direction="row" alignItems="center">
+                  <Grid item xs={6}>
+                    <h1>Shopping List:</h1>
+                  </Grid>
+                  <Grid item xs={6}>
+                    <TextInputField
+                      text={curBarcode ? curBarcode : "...barcode"}
+                      setState={setDebugItem}
+                    />
+                  </Grid>
+                </Grid>
+                {shoppingList.length > 0 && (
+                  <Grid
+                    container
+                    spacing={1}
+                    direction="column"
+                    alignItems="stretch"
+                  >
+                    <Grid
+                      container
+                      direction="row"
+                      justify="center"
+                      alignItems="center"
+                    >
+                      <Grid item xs={3}></Grid>
+                      <Grid item xs={3}>
+                        <Typography variant="subtitle1">Name</Typography>
+                      </Grid>
+                      <Grid item xs={2}>
+                        <Typography variant="subtitle1">Unit Price</Typography>
+                      </Grid>
+                      <Grid item xs={2}>
+                        <Typography variant="subtitle1">Subtotal</Typography>
+                      </Grid>
+                      <Grid item xs={2}></Grid>
+                    </Grid>
+                    <Divider />
+                    {renderShoppingList()}
+                  </Grid>
+                )}
+              </Box>
+            </Paper>
+          </Grid>
+        )}
+        <button onClick={testBarcode}>Test Barcode API</button>
+        <img
+          id="testImgSrc"
+          hidden={true}
+          width="200"
+          height="200"
+          src={SampleBarcode}
+        />
+        {uploadImg.mimeType && (
+          <Grid item xs={12} justify="center">
+            <Paper elevation={2}>
+              <img
+                id="uploadImgSrc"
+                hidden={!debugImg}
+                width="200"
```

Comment:
I am actually not sure whether there would be accessibility issue if we just absolute value for the width/height. We can create a issue to track possible accessibility concerns.

Reply:
This is for debugging purposes, and will not be shown to user

Reply:
#30

Reply:
good reminder on accessibility, created a new issue #55 for the future